### DISCUSSION TITLE
Creating Update site without zip

### DISCUSSION THREAD

**Discussion by vogella** [eclipse-tycho/tycho#4439] [Q&A] [answered]
Currently Tycho will create a p2 repository for an update site project, which is create. It also creates a zip file for the this repo.

![image](https://github.com/user-attachments/assets/d43f7eab-d12c-4da1-84e0-a4ed8ae9e89a)

Is it possible to instruct Tycho to not create the additional zip file?

While not breaking anything, the creation of the zip file is not always needed and just adds additional build time to the build.

**Comment by laeubi** [comment, marked as answer]
https://tycho.eclipseprojects.io/doc/latest/tycho-p2-repository-plugin/archive-repository-mojo.html#skipArchive

**Reply by vogella** [reply]
Thanks!
